### PR TITLE
Cellpack tweaks

### PR DESCRIPTION
**Pull request by arose**
- Add ``instanceGranularity`` option for marker, transparency, clipping, overpaint, substance data to save memory
- CellPack extension tweaks
    - Use instancing to create DNA/RNA curves to save memory
    - Enable ``instanceGranularity`` by default

### CODE REVIEW

**Comment by arose** [comment]
Hey @corredD here are some cellpack updates. With this you can render the ISG without errors. Example https://www.dropbox.com/sh/sbhdy724vziriv5/AAAFe2W4VKtnQgXl_TIs7koha?dl=0

**Comment by arose** [comment]
Hey @corredD here are some cellpack updates. With this you can render the ISG without errors. Example https://www.dropbox.com/sh/sbhdy724vziriv5/AAAFe2W4VKtnQgXl_TIs7koha?dl=0.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ Note that since we don't clearly distinguish between a public and private interf
 
 ## [Unreleased]
 
+- Add ``instanceGranularity`` option for marker, transparency, clipping, overpaint, substance data to save memory
+- CellPack extension tweaks
+    - Use instancing to create DNA/RNA curves to save memory
+    - Enable ``instanceGranularity`` by default
```
Comment (dsehnal) [inline CHANGELOG.md:12]: I think this seems to be missing here. I tried looking for ``instanceGranuality`` in the ``cellpack`` folder and only found ``forceInstanceTheme: true``, but this property doesn't seem to be actually read anywhere.

I also tried to load some cellpack models and it still seems to use the "group" granuality. Am I missing something?

Comment (arose) [reply CHANGELOG.md:12]: yeah, thanks, missed that when renaming forceInstanceTheme -> instanceGranularity

Comment (arose) [reply CHANGELOG.md:12]: thinking about making instanceGranularity a quality param

Comment (dsehnal) [reply CHANGELOG.md:12]: That might be a good improvement. What would the threshold be?

**Review by dsehnal — APPROVED** [review]
Very cool, works now.

File: CHANGELOG.md (hunk 1 above)
Comment (arose) [reply CHANGELOG.md:12]: Thought about it a bit and decided not to handle it as a quality param. The current quality levels handle systems up to 1,000,000 atoms, everything beyond gets generally the same quality. The ``instanceGranularity`` settings only really starts to make sense for systems with 100,000,000 atoms and more. In short it is quite specific so for now it should be set manually in specific context like for cellpack models.

**Pull request merged**